src/styles/page-template/index.md.njk

```diff
@@ -243,6 +243,14 @@ To change the components that are included in the page template by default, set
       </td>
     </tr>
 
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">scriptNonce</td>
+      <td class="govuk-table__cell">Variable</td>
+      <td class="govuk-table__cell">
+        Add a nonce attribute to the script for your Content Security Policy (CSP). Provide a nonce that hostile actors cannot guess, as otherwise they can easily find a way around your CSP.
```

Comment:
Is it a nonce attribute or a nonce variable?

Reply:
```suggestion
        Add a nonce attribute to inline scripts for your Content Security Policy (CSP). Provide a nonce that hostile actors cannot guess, as otherwise they can easily find a way around your CSP.
```

Reply:
@36degrees I tried not to make ^ overlong. For accessibility, the advice is to be specific about the location of anchor links.

Reply:
Can we use the same link text we're using in the release notes? It seems slightly more targeted.

```suggestion
        Add a nonce attribute to inline scripts for your Content Security Policy (CSP). Provide a nonce that hostile actors cannot guess, as otherwise they can easily find a way around your CSP. [Read more about how to configure your CSP to work with GOV.UK Frontend](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#select-and-initialise-part-of-a-page)
```

Reply:
```suggestion
        Add a nonce attribute to the script for your Content Security Policy (CSP). Provide a nonce that hostile actors cannot guess, as otherwise they can easily find a way around your CSP. However, you should use this option only if you're not able to [include the hash for the inline scripts in your service's Content Security Policy (CSP)](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#select-and-initialise-part-of-a-page).```

Reply:
@36degrees Should 'nonce attribute' be '`nonce` attribute'?